robusta_krr/core/runner.py

```diff
@@ -32,6 +32,9 @@ def custom_print(*objects, rich: bool = True, force: bool = False) -> None:
         print_func(*objects)  # type: ignore
 
 
+class CriticalRunnerException(Exception): ...
```

Comment:
what is ... after a class creation?

Reply:
Short answer: It is kinda the same as to write `pass` in this context
Long: https://stackoverflow.com/questions/772124/what-does-the-ellipsis-object-do